src/commands/Social/Member Management/social.ts

```diff
@@ -24,29 +27,66 @@ export default class extends SkyraCommand {
 	}
 
 	public async add(message: KlasaMessage, [user, amount]: [KlasaUser, number]) {
-		const newAmount = await this.client.queries.upsertIncrementMemberSettings(message.guild!.id, user.id, amount);
-		return message.sendLocale('COMMAND_SOCIAL_ADD', [user.username, newAmount, amount]);
+		const connection = await DbSet.connect();
```

Comment:
Destructure `members` from the object
```suggestion
		const { members } = await DbSet.connect();
```